ui/overview_page.go

```diff
@@ -309,55 +271,185 @@ func (pg *overviewPage) centralize(gtx layout.Context, content layout.Widget) la
 	)
 }
 
-// recentTransactionRow lays out a single row of a recent transaction.
-func (pg *overviewPage) recentTransactionRow(gtx layout.Context, txn transactionWidgets, c pageCommon) layout.Dimensions {
-	gtx.Constraints.Min.X = gtx.Constraints.Max.X
-	return layout.Flex{Axis: layout.Horizontal, Spacing: layout.SpaceBetween}.Layout(gtx,
-		layout.Rigid(func(gtx C) D {
-			return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
-				layout.Rigid(func(gtx C) D {
-					return layout.Inset{Top: values.MarginPadding5}.Layout(gtx, txn.direction.Layout)
-				}),
-				layout.Rigid(func(gtx C) D {
-					return layout.Inset{Left: values.MarginPadding15, Top: values.MarginPadding5}.Layout(gtx, func(gtx C) D {
-						return layoutBalance(gtx, txn.balance, c)
-					})
-				}),
-			)
-		}),
-		layout.Rigid(func(gtx C) D {
-			return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+// recentTransactionRows lays out rows of a recent transaction.
+func (pg *overviewPage) recentTransactionRows(c pageCommon) []layout.Widget {
+	walletNameVisibility := len(c.info.Wallets) < 1
+	var transactionRows []layout.Widget
+	theme := pg.theme
+	for index, txn := range (*pg.walletTransactions).Recent {
+		txnWidgets := transactionWidgets{
+			wallet:  theme.Body1(txn.WalletName),
+			balance: txn.Balance,
+			date:    theme.Body1(txn.DateTime),
+			status:  theme.Body1(""),
+		}
+		if txn.Txn.Direction == dcrlibwallet.TxDirectionSent {
+			txnWidgets.direction = c.icons.sendIcon
+		} else {
+			txnWidgets.direction = c.icons.receiveIcon
+		}
+
+		if txn.Status == "confirmed" {
+			txnWidgets.status.Text = formatDateOrTime(txn.Txn.Timestamp)
+			txnWidgets.statusIcon = c.icons.confirmIcon
+		} else {
+			txnWidgets.status.Text = txn.Status
+			txnWidgets.statusIcon = c.icons.pendingIcon
+		}
+
+		txnWidgets.direction.Scale = 1
+		txnWidgets.statusIcon.Scale = 1
+
+		click := pg.toTransactionDetails[index]
+
+		row := func(gtx C) D {
+			pointer.Rect(image.Rectangle{Max: gtx.Constraints.Max}).Add(gtx.Ops)
+			click.Add(gtx.Ops)
+			return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
 				layout.Rigid(func(gtx C) D {
-					return layout.Inset{Top: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
-						return txn.status.Layout(gtx)
+					heightWithResolution := pg.txnRowHeight * int(gtx.Metric.PxPerDp)
+					gtx.Constraints.Min.Y, gtx.Constraints.Max.Y = heightWithResolution, heightWithResolution
```

Comment:
kindly add a comment to code like this that's not a common UI layout or modification